backend/src/handle_patients.py

```diff
@@ -0,0 +1,60 @@
+"""Provide an endpoint to retrieve all current patients for a station."""
+import datetime
+from django.http import JsonResponse
+from ..models import Patient, DailyClassification
+from django.db.models import Case, When, BooleanField
+
+
+def get_patients_per_station(station_id: int) -> list:
+    """Get all patients assigned to a station and add a classified parameter.
+
+    The classified parameter is set to True if the patient has already been classified today.
+    A patient is assigned to a station if the patient has been classified for the station yesterday or today.
+    This is because we have no access to the internal database of the hospital.
+
+    Args:
+        station_id (int): The ID of the station in the database.
+
+    Returns:
+        list: The patients assigned to the station.
+    """
+    today = datetime.date.today()
+    yesterday = today - datetime.timedelta(days=1)
+
+    # Query DailyClassification for the relevant time range and station
+    classifications = DailyClassification.objects.filter(
+        station_id=station_id,
+        classification_date__in=[yesterday, today]
```

Comment:
Not sure if we should take a classification from today or yesterday for granted. IIRC during hospitation I saw that a station did not classifiy patients for multiple days straight

Reply:
In that case the line below triggers a return where everything is set to false.
See line 31.

Reply:
In the case that the last classification date of a patient is older than yesterday, the patient will not appear on that station.
I dont really see another option (yet), since the classification is the only interaction with the database so far.
Lmk if you have a suggestion.